scss/typography/_helpers.scss

```diff
@@ -38,6 +38,29 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+/// @dropcaps
+
+// Line height for all the text within dropcaps, disabling default line height's
+$dropcaps-line-height : 1.45 !default;
+
+// @dropcaps-first-letter
+
+// Lets set margin first, used tiny negative margins for proper alignment
+$dropcaps-first-letter-margin-top: 0.02em !default;
+$dropcaps-first-letter-margin-bottom: -0.05em !default;
+
+// Padding Right to the dropcaps first letter to create gap b/w rest of text 
+$dropcaps-first-letter-padding-right: 0.05em !default;
+
+// Dropcaps first charchter is 500% bigger then regular size of an element 
+$dropcaps-first-letter-font-size: $global-font-size * 5 !default;
+
+// Line height for first letter of the dropcaps
+$dropcaps-first-letter-line-height: 0.85em !default;
+
+// Dropcaps float
+$dropcaps-first-letter-float: left;
```

Comment:
We can only use `$global-left` without options. There is no case where you would want to have the first letter on the right with a LTR context.